pkg/controllers/membercluster/membercluster_controller.go

```diff
@@ -251,8 +255,15 @@ func (r *Reconciler) syncNamespace(ctx context.Context, mc *fleetv1alpha1.Member
 		return namespaceName, nil
 	}
 
-	// TODO: Update namespace if currentNS != expectedNS.
-
+	// migration: To add new label to all existing member cluster namespaces.
+	klog.V(2).InfoS("patching namespace", "memberCluster", klog.KObj(mc), "namespace", namespaceName)
```

Comment:
Hi Arvind! Just a nit here, but would you mind adding a check here to make sure that the patch will not happen if the label is already present?

I understand that patch will not happen if the patch data bytes are empty, but the log emitted here might be a little confusing.